Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -978,7 +982,12 @@ def self.resource_version(
           },
         }
 
-        resource_version_info[:meta] = { livecheckable: has_livecheckable, url: {} }
+        resource_version_info[:meta] = {
+          has_livecheck_block: has_livecheck_block,
+          # TODO: Remove `livecheckable` field after aliases are removed
+          livecheckable:       has_livecheck_block,
```

Comment:
Similar to going through a deprecation phase with `#livecheckable?`, this keeps the `livecheckable` field around for the time being. That said, this is a different situation, as there's nothing to make it clear that the field name has changed to `has_livecheck_block`. This simply delays the point when the field will be removed and third-party code will break, so it may not make as much sense here.

As above, I have some personal projects that parse livecheck JSON output and I'll need to update those to handle the new `has_livecheck_block` field name (falling back to the older `livecheckable` name if not found) but that's not a problem for me.

Reply:
Yeah, this is one of the difficult things with the JSON representation since it's public but we can't smoothly deprecate anything. My 2 cents are that it should be fine to remove since it's probably not something people rely upon much assuming that it's communicated in the release notes.

Reply:
I removed the `livecheckable` field from the JSON output in the latest push, so we will only have `obj[:meta][:livecheck_defined]` going forward 👍